libecs/include/pivot/ecs/Core/types.hxx

```diff
@@ -2,16 +2,18 @@
 
 #include <bitset>
 
+#include <pivot/utility/entity.hxx>
+
+using Entity = pivot::Entity;
+const Entity MAX_ENTITIES = pivot::MAX_ENTITIES;
+
 constexpr std::uint32_t fnv1a_32(char const *s, std::size_t count)
 {
     return ((count ? fnv1a_32(s, count - 1) : 2166136261u) ^ s[count]) * 16777619u;    // NOLINT (hicpp-signed-bitwise)
 }
 
 constexpr std::uint32_t operator"" _hash(char const *s, std::size_t count) { return fnv1a_32(s, count); }
```

Comment:
Dear lord, the f** is this ? If it is not in use can you remove it ? 🤯

Reply:
That's your code :grimacing: